_topic_map.yml

```diff
@@ -1021,35 +1021,38 @@ Topics:
   - Name: Container-native Virtualization 2.0 release notes
     File: cnv-release-notes
 #---
```

Comment:
@joaedwar, I believe this line needs to be uncommented; It's part of the YAML structure and without it, the Travis build breaks.